src/compile/compile.ts

```diff
@@ -48,7 +48,7 @@ export function compileRootGroup(model: Model) {
   const spec = model.spec();
 
   let rootGroup:any = extend({
-      name: spec.name ? spec.name + '-root' : 'root',
+      name: spec.name ? spec.name + '-rootCell' : 'rootCell',
```

Comment:
This is good to know haha.  Are there other reserved names that I should be aware of?  (Maybe we should put this in Vega's docs.) 

Btw, how about `layout-root`?  Because this isn't really a cell as in unit's cell.